File: Problem Solving/C/arrayinsertarray.c
/*inserting one array element int another array*/
#include<stdio.h>
#include<conio.h>
void main()
{
int a[20],b[20],asize,bsize,loc;
printf("enter the size of a array\n");
scanf("%d",&asize);
for(int i=0;i<asize;i++)
{
scanf("%d",&a[i]);
}
for(int i=0;i<asize;i++)
{
printf("%d\n",a[i]);
}
printf("enter the size of b array\n");
scanf("%d",&bsize);
for(int i=0;i<bsize;i++)
{
scanf("%d",&b[i]);
}
for(int i=0;i<bsize;i++)
{
printf("%d\n",b[i]);
}
printf("enter the location to insert the array");
scanf("%d",&loc);
for(int i=asize-1;i>=loc;i--)
{
a[i+bsize]=a[i];
}
for(int i=0;i<bsize;i++)
{
a[i+loc]=b[i];
}
for(int i=0;i<bsize+asize;i++)
{
printf("%d",a[i]);
}
getch();
}

File: Problem Solving/Python/arrayinsertarray.py
#program to insert an array at specific index in another array
from array import *
size1=int(input("enter the size of the first array"))
arr1=array('i',[])
print("enter the first array elements")
for i in range(size1):
    value1=int(input())
    arr1.append(value1)
print(arr1)
size2=int(input("enter the size of the second array"))
arr2=array('i',[])
print("enter the second array elements")
for i in range(size2):
    value2=int(input())
    arr2.append(value2)
print(arr2)
location=int(input("Enter the location where you want to insert element"))
for i in range(0,size2):
    arr1.insert(location,arr2[i])
    location=location+1
print(arr1)    
